### ISSUE TITLE
DIM Weapon perks seem to be smaller

### ISSUE THREAD

**Issue by Ate75** [DestinyItemManager/DIM#9481]
### Proposed change

Even when I change to grid,then compare the perks,they are too small to see well ,this is different than recently,can we adj the size in settings? 

### How does this fit into your workflow?

I need to expand every perk on every gun now instead of seeing it larger and knowing what it is

**Comment by robojumper** [comment]
Presumably a regression from #9463 -- this was reported on Discord too

![screenshot](https://media.discordapp.net/attachments/333464468241514497/1111170619573284874/image0.jpg)

This appears to only happen in the grid view of perks, not the list view.

**Comment by Ate75** [comment]
When you compare items it uses the grid view with
 the smaller perk views

**Comment by bhollis** [comment]
Yup, just a miss from the CSS changes when in mobile mode. Feel free to use the release version until this is fixed
